portfolio/src/main/webapp/comment.html

```diff
@@ -22,7 +22,17 @@
           <button type="submit">Submit</button>
         </form>
       </section>
-      <section id="comment-display" class="comment-feed-container"></section>
+      <section class="comment-feed-container">
+        <form class="container radio" method="POST">
+          <label for="three">3</label>
+          <input type="radio" id="three" name="quantity" value="3" onchange="updateComments(this)" checked>
```

Comment:
`(this)` param not needed, you can get it from the event.

```
onchange="updateComments"
```

Reply:
removed